purchase_deposit/wizard/purchase_make_invoice_advance.py

```diff
@@ -169,6 +169,23 @@ def create_invoices(self):
             amount = self.amount
             if self.advance_payment_method == "percentage":  # Case percent
                 amount = self.amount / 100 * order.amount_untaxed
+            # calculate all deposit lines
+            sum_deposit = (
+                sum(
+                    line.price_unit
```

Comment:
```suggestion
                    line.price_unit
```
I'm not sure price_unit is include / exclude tax.
Maybe we should have config to check include or exclude tax. What do you think?